vissl/trainer/train_task.py

```diff
@@ -322,6 +322,9 @@ def build_dataloaders(
 
         return loaders
 
+    def num_training_iterations(self):
+        return len(self.dataloaders["train"]) * self.config["OPTIMIZER"]["num_epochs"]
```

Comment:
Excellent :) I love that !

Reply:
nit: add an assert to check that the self.dataloaders is created and defined. Also , sometimes, users can run "TEST_ONLY" mode in VISSL so the "train" will not exist. We should handle the assumptions via asserts.